jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/SlidersTests.java

```diff
@@ -8,27 +8,42 @@
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
+import java.util.List;
+import java.util.stream.Collectors;
 import java.util.stream.IntStream;
+import java.util.stream.Stream;
 
 import static com.jdiai.tools.Timer.waitCondition;
 import static io.github.com.StaticSite.slidersPage;
 import static io.github.com.enums.Colors.BLUE_ACCENT_5;
 import static io.github.com.enums.Colors.ORANGE_DARKEN_3;
+import static io.github.com.pages.SlidersPage.adjustableSlider;
 import static io.github.com.pages.SlidersPage.appendTextFieldInput;
 import static io.github.com.pages.SlidersPage.appendTextFieldSlider;
 import static io.github.com.pages.SlidersPage.colorsSlider;
+import static io.github.com.pages.SlidersPage.denseSwitch;
 import static io.github.com.pages.SlidersPage.disabledSlider;
 import static io.github.com.pages.SlidersPage.disabledSliderControl;
 import static io.github.com.pages.SlidersPage.discreteSlider;
+import static io.github.com.pages.SlidersPage.errorCountTextField;
+import static io.github.com.pages.SlidersPage.heightTextField;
+import static io.github.com.pages.SlidersPage.hideDetailsSwitch;
 import static io.github.com.pages.SlidersPage.iconsSlider;
+import static io.github.com.pages.SlidersPage.inverseLabelSlider;
+import static io.github.com.pages.SlidersPage.loaderHeightTextField;
+import static io.github.com.pages.SlidersPage.loadingSwitch;
 import static io.github.com.pages.SlidersPage.readonlySlider;
 import static io.github.com.pages.SlidersPage.stepSlider;
+import static io.github.com.pages.SlidersPage.successMessageTextField;
+import static io.github.com.pages.SlidersPage.themeSwitch;
 import static io.github.com.pages.SlidersPage.thumbSlider;
 import static io.github.com.pages.SlidersPage.thumbSliderControl;
 import static io.github.com.pages.SlidersPage.ticksSlider;
+import static io.github.com.pages.SlidersPage.validateOnBlurSwitch;
 import static io.github.com.pages.SlidersPage.validationSlider;
 import static io.github.com.pages.SlidersPage.verticalSlidersSlider;
 import static org.openqa.selenium.Keys.BACK_SPACE;
+import static org.openqa.selenium.Keys.ENTER;
 
 public class SlidersTests extends TestsInit {
 
```

Comment:
1) Please, check that in every test there is mmmm.show()
2) Do we need this method?
private void clearInputField(UIElement inputField) {
        IntStream.range(1, 4).forEach(i -> inputField.sendKeys(BACK_SPACE));
    }